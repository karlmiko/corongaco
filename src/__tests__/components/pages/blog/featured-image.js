import React from 'react'
import renderer from 'react-test-renderer'
import FeaturedImage from '~components/pages/blog/featured-image'

describe('Components : Pages : Blog : Featured Image', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <FeaturedImage
          image={{
            resize: {
              src: '//images.ctfassets.net/anexampleimage',
            },
            title: 'An Example Image',
          }}
        />,
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
