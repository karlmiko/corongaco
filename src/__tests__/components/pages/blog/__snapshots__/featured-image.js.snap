// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Pages : Blog : Featured Image renders correctly 1`] = `
Array [
  <div>
    <img
      alt="An Example Image"
      src="//images.ctfassets.net/anexampleimage"
    />
    <div
      className="credit"
    >
      An Example Image
    </div>
  </div>,
  <hr
    className="featuredImageRule"
  />,
]
`;
